### DISCUSSION TITLE
Deploy LocalAI in LXC container

### DISCUSSION THREAD

**Discussion by mate1213** [mudler/LocalAI#1505] [Q&A]
I try to deploy LocalAI from source in LXC container.
But I'm stuck with the following error:
```
CMake Error at examples/grpc-server/CMakeLists.txt:26 (find_package):
  Could not find a package configuration file provided by "Protobuf" with any
  of the following names:

    ProtobufConfig.cmake
    protobuf-config.cmake
```

I tried to install `libprotobuf-dev` and `protobuf-compiler` but no luck.
I use `Debian 12`.

**Comment by manuelkamp** [comment]
use ubuntu 22.04 LXC container it works there

**Reply by mate1213** [reply]
Thank you very much! I will try.

**Reply by mate1213** [reply]
I tried out in Ubuntu 22.04 updated version, I've got the same error.
![image](https://github.com/mudler/LocalAI/assets/36986834/a5d572ff-cabc-4cbb-8836-6422a045d31e)

**Comment by chris-hatton** [comment]
Same error here, attempting build on Ubuntu 24.
